### PR TITLE
Hide 'Copy' button before a seed phrase is generated

### PR DESCRIPTION
**Pull request by keshan3262**
Screenshot_1623927801
Screenshot_1623927814

### CODE REVIEW

File: src/components/mnemonic/mnemonic-create/mnemonic-create.tsx [hunk 1]
```diff
@@ -45,8 +45,12 @@ export const MnemonicCreate: FC<MnemonicProps> = ({ value, isError, onChangeText
       />
       <View style={styles.buttonsContainer}>
         <ButtonSmallSecondary title="GEN NEW" onPress={handleGenerateNewButtonPress} />
-        <Divider size={formatSize(8)} />
-        <ButtonSmallSecondary title="COPY" onPress={() => copyStringToClipboard(value)} />
+        {value ? (
```
Comment (unixvb) [inline src/components/mnemonic/mnemonic-create/mnemonic-create.tsx:48]: `{isDefined(value) && <>...</>}`

Comment (keshan3262) [reply src/components/mnemonic/mnemonic-create/mnemonic-create.tsx:48]: I'll check it a bit later... As far as I remember, the initial value is an empty string, so `isDefined` doesn't work here properly

Comment (keshan3262) [reply src/components/mnemonic/mnemonic-create/mnemonic-create.tsx:48]: Screenshot_2021-06-17_14-41-37
Sorry, a single `isDefined` check won't work here properly
Then do we really need here something like `isDefined(value) && value.length > 0`?

Comment (unixvb) [reply src/components/mnemonic/mnemonic-create/mnemonic-create.tsx:48]: so, use `isString` istead of `isDefined` (it checks for empty string value)

**Review by unixvb — APPROVED** [review]

**Pull request merged**